src/App.jsx

```diff
@@ -14,57 +17,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const showSelection = selectedGood
+    ? `${selectedGood} is selected`
+    : 'No goods selected';
+  const setSelection = currentGood =>
+    selectedGood === currentGood
+      ? setSelectedGood('')
+      : setSelectedGood(currentGood);
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {showSelection}
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+        {!!selectedGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => {
+              setSelectedGood('');
+            }}
```

Comment:
Сreate a helper function for reset logic and use it everywhere
```suggestion
            onClick={resetSelectedGood}
```